e2e/export_import_deprecated/test_online_export_import/test.yml

```diff
@@ -0,0 +1,12 @@
+- query: >-
+    MATCH (n) RETURN count(n) AS count_after_export;
+  output: 
+    - count_after_export: 3
+- query: >-
+    MATCH (n) RETURN count(n) AS count;
+  output: 
+    - count: 0
+- query: >-
+    MATCH (n) RETURN count(n) AS count_after_import;
+  output:
+    - count_after_import: 3
```

Comment:
Sorry to bother, but could you write another test with other data types included. For example, having time data, null, list, or dict. Just to see if it works with complex datatypes

Reply:
It's great that you asked this, since I totally forgot about temporal types. I need to work on this a bit more.

Reply:
Info on my discovery so far:
- I think that there is no problem with lists and I added test to support that.
- I think that `null` is okay. If it's added as a property in the import_file, then it won't be created in the database, just like when running a Cypher query with a property set to `null`. This means that later on it won't be exported with export_util, but then I think that we don't have to even bother with null types? (since we are expecting that the user imports only file from the export module, as said in the documentation).
- I added a test for the dictionary as a part of the property, and that should also work fine.
- Temporal types are currently an issue and I will continue working on them next week.

Reply:
Just drop a message after investigation on temporal types :smile: